src/Drawer/Drawer.spec.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-env mocha */
+import React from 'react';
+import {shallow} from 'enzyme';
+import consoleError from '../../test/utils/consoleError';
+import Drawer from './Drawer';
+import getMuiTheme from '../styles/getMuiTheme';
+
+describe('<Drawer />', () => {
+  const muiTheme = getMuiTheme();
+  const shallowWithContext = (node) => shallow(node, {context: {muiTheme}});
+
+  describe('propTypes', () => {
+    before(() => {
+      consoleError();
```

Comment:
Sorry, I would have provided more detail. I was suggesting it injecting it in the [setup](https://github.com/callemall/material-ui/blob/master/test/index.js) so every tests benefits from it.